web/src/components/LocationLayout.tsx

```diff
@@ -28,6 +31,8 @@ type PropsType = {
 
 const LocationLayout = (props: PropsType): ReactElement => {
   const [feedbackModalRating, setFeedbackModalRating] = useState<FeedbackRatingType | null>(null)
+  const theme = useTheme()
+  useScript('https://unpkg.com/@rasahq/rasa-chat')
```

Comment:
We should add a condition here, since for the moment we only want to load the script in devFriendlyMode & for Munich but the script will be loaded in any case even its not rendered in the DOM